other/docker/windows/build_toxcore.sh

```diff
@@ -93,26 +106,35 @@ build()
     cp "${STATIC_TOXCORE_PREFIX_DIR}"/* "${RESULT_PREFIX_DIR}" -R
     cp "${DEP_PREFIX_DIR}"/* "${RESULT_PREFIX_DIR}" -R
 
-    # make libtox.dll
-    cd "${SHARED_TOXCORE_PREFIX_DIR}"
-    for archive in ${STATIC_TOXCORE_PREFIX_DIR}/lib/libtox*.a
-    do
-        ${WINDOWS_TOOLCHAIN}-ar xv ${archive}
-    done
-    ${WINDOWS_TOOLCHAIN}-gcc -Wl,--export-all-symbols \
-                             -Wl,--out-implib=libtox.dll.a \
-                             -shared \
-                             -o libtox.dll \
-                             *.obj \
-                             ${STATIC_TOXCORE_PREFIX_DIR}/lib/*.a \
-                             ${DEP_PREFIX_DIR}/lib/*.a \
-                             /usr/${WINDOWS_TOOLCHAIN}/lib/libwinpthread.a \
-                             -liphlpapi \
-                             -lws2_32 \
-                             -static-libgcc
-    cp libtox.dll.a ${RESULT_PREFIX_DIR}/lib
-    mkdir -p ${RESULT_PREFIX_DIR}/bin
-    cp libtox.dll ${RESULT_PREFIX_DIR}/bin
+    # failed on Cygwin due to undefined reference to `sysconf` in libvpx.a
+    if [ "${CROSS_COMPILE}" = "true" ]; then
+        # make libtox.dll
+        cd "${SHARED_TOXCORE_PREFIX_DIR}"
+        for archive in ${STATIC_TOXCORE_PREFIX_DIR}/lib/libtox*.a
+        do
+            ${WINDOWS_TOOLCHAIN}-ar xv ${archive}
+        done
+        if [ "${CROSS_COMPILE}" = "true" ]; then
```

Comment:
We are already in a `if [ "${CROSS_COMPILE}" = "true" ]; then` block, why check it again?

Reply:
Yeah, you're right.
I forgot to clean this up when I was trying to get `libtox.dll` thing work.